services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/PortHistorySpec.groovy

```diff
@@ -150,13 +155,69 @@ class PortHistorySpec extends HealthCheckSpecification {
         }
     }
 
+    def "Port history is able to show ANTI_FLAP statistic"() {
+        given: "A direct link"
+        def isl = getTopology().islsForActiveSwitches.find { !it.aswitch }
+        assumeTrue("Unable to find ISL for this test", isl as boolean)
+        def timestampBefore = System.currentTimeMillis()
+
+        when: "Execute port DOWN on the src switch"
+        antiflap.portDown(isl.srcSwitch.dpId, isl.srcPort)
+        Wrappers.wait(WAIT_OFFSET / 2) {
+            assert islUtils.getIslInfo(isl).get().state == IslChangeType.FAILED
+        }
+
+        then: "Port history is created on the src switch"
+        Wrappers.wait(WAIT_OFFSET) {
+            Long timestampAfterDown = System.currentTimeMillis()
+            with(northboundV2.getPortHistory(isl.srcSwitch.dpId, isl.srcPort, timestampBefore, timestampAfterDown)) {
+                assert it.size() == 2
+            }
+        }
+
+        when: "Generate antiflap statistic"
+        def count = 0
+        //confd/templates/wfm/topology.properties.tmpl =>  port.antiflap.stats.dumping.interval.seconds = 60
+        def antiflapDumpingInterval = 60
```

Comment:
I don't understand what this interval means. Can you please explain in a comment?